openmdao/core/system.py

```diff
@@ -1539,16 +1507,13 @@ def _setup_vec_names(self, mode):
             for system in self.system_iter(include_self=True, recurse=True):
                 for name, meta in system._get_vec_names_from_vois(mode):
                     vois.add(system.get_source(name))
-                    if meta['vectorize_derivs']:
```

Comment:
Should the argument `vectorized_derivs` also be removed from add_design_var, etc? 

Note: there is also use and mention of this feature in the docs, but that will have to be removed after the new notebooks are merged in.

Reply:
Removed all refs to 'vectorize_derivs'.